src/vcf/validator.cpp

```diff
@@ -254,12 +254,24 @@ namespace ebi
             { { 120, -100 }, ZLIB }
         };
 
+        /*
+         * If the first line of the VCF file has less than 5 characters,
+         * std::equal will cause a segmentation fault.
+         */
+        if (line.size() < 5) {
+            if (line.size() == 0) {
+                BOOST_LOG_TRIVIAL(warning) << "Empty vcf file";
```

Comment:
put VCF in uppercase, and write a message that adds more context, like "The VCF file provided is empty"